scripts/auto_dock.py

```diff
@@ -14,127 +14,191 @@
 from geometry_msgs.msg import TwistStamped
 from geometry_msgs.msg import Transform
 from fiducial_msgs.msg import FiducialTransformArray
-#from tf.msgs import tf
+from wibotic_msg.srv import ReadParameter
+
 from tf.transformations import *
 
-class ArucoDockingManager(object):
-    MANAGER_PERIOD = 0.1
-    CMD_VEL_ANGULAR_RATE = 0.75 #rad/s negative is clockwise
-    CMD_VEL_LINEAR_RATE = 0.5 #m/s
-    TURN_RADIANS = -1.0472/2 #not exact
-    TURN_DURATION = abs(TURN_RADIANS/CMD_VEL_ANGULAR_RATE)
-    MIN_TURN_PERIOD = 0.18
-    MAX_RUN_TIMEOUT = 240 #in seconds
-    ARUCO_SLOW_WARN_TIMEOUT = rospy.Duration(1) #in seconds
-    ARUCO_WAIT_TIMEOUT = 2 #in seconds
-
-    CANCELLED_TIMEOUT = 10 #in seconds
-    START_DELAY = 2.0
-
-    APPROACH_ANGLE = 0.1
-    Z_TRANS_OFFSET = 0 #0.5
-    #K_P = 1.5
-    ARUCO_CALLBACK_COUNTER_MAX = 5
-
-    JOG_DISTANCE = 0.3
-    FINAL_APPROACH_DISTANCE = 1.0
-    WIGGLE_RADIANS = -0.5
-    DOCK_ARUCO_NUM = 0
-    UNDOCK_DISTANCE = 1.0
-
-    MAX_CENTERING_COUNT = 50
 
+class ArucoDockingManager(object):
```

Comment:
Just a note if you're interested. These variables below are static variables. 

Best practice for class global variables is using CAPS_LOCK, to show variables that won't change kinda like constants.

Another type of these variables defined below a class is a static variable with snake_case. These variables are shared throughout all instances (objects) of this class even when they are changed.

Reply:
yeah, thanks!  i left it the way they had it even though it's weird.  I'm going to change it.  

Since they're coming from rosparam, they can't be caps locked global variables, so I'll just make them all snake_case if that makes the most sense to you

Reply:
i made a bunch of formatting changes, lmk what you think!